seahorn/jobs/hash_table_foreach/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_executable(hash_table_foreach
+  ${AWS_C_COMMON_ROOT}/source/common.c
+  aws_hash_table_foreach_harness.c)
+set(MAX_TABLE_SIZE 2)
```

Comment:
I think the `MAX_TABLE_SIZE` bound and `UNROLL_BOUND` can be relaxed for the non-deep version of the verification task.